DXMainClient/Program.cs

```diff
@@ -100,6 +102,9 @@ static void InitializeApplicationConfiguration()
 #endif
         static void Main(string[] args)
         {
+            // https://stackoverflow.com/questions/3967716/how-to-find-encoding-for-1251-codepage
+            Encoding.RegisterProvider(CodePagesEncodingProvider.Instance);
```

Comment:
This line was executed in `ClientCore.PlatformShim.EncodingExt`. Since that static initialization method does not guarantee to be executed, I guess it is okay to execute it in the main method.

Reply:
Done by 2336d2f74966e9a646000cb504ac2cf7d66af953.

Reply:
I am not suggesting changes in this comment